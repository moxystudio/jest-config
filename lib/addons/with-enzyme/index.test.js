'use strict';

const withEnzyme = require('.');

it('should match snapshot', () => {
    expect(withEnzyme('enzyme-adapter-react-16')()).toMatchSnapshot();
});

it('should throw if no adapter is passed', () => {
    expect(() => withEnzyme()).toThrow(/please provide an enzyme adapter/i);
});
