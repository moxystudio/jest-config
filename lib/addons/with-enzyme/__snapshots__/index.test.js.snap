// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match snapshot 1`] = `
Object {
  "globals": Object {
    "__MOXY_JEST_CONFIG_ENZYME_ADAPTER__": "enzyme-adapter-react-16",
    "foo": "bar",
  },
  "setupFilesAfterEnv": Array [
    "foo",
    "<PROJECT_ROOT>/lib/addons/with-enzyme/setup-adapter.js",
    "<PROJECT_ROOT>/lib/addons/with-enzyme/setup.js",
  ],
}
`;
